### DISCUSSION TITLE
请教下 在umi3/4中怎么使用、reduxjs/toolkit?????

### DISCUSSION THREAD

**Discussion by t1z001** [umijs/umi#10169] [Show and tell]
请教下 在umi3/4中怎么使用、reduxjs/toolkit?????

**Comment by fz6m** [comment]
在 umi 4 中，可以参考 https://github.com/umijs/umi/tree/master/examples/with-redux-toolkit 这个例子。

重点是如何创建 Provider ，使用方法自行决定。
